### PR TITLE
Add sainnhe/gruvbox-material

### PR DESCRIPTION
**Pull request by adam-devel**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -2374,6 +2374,13 @@
             "summary": "A tree-sitter grammar for nu-lang, the language of nushell",
             "license": "MIT",
             "rockspec_template": "nurr/resources/tree-sitter-nu.template"
+        },
+        {
+            "name": "sainnhe/gruvbox-material",
+            "shorthand": "gruvbox-material",
+            "dependencies": [],
+            "summary": "Gruvbox with Material Palette",
```
Comment (mrcjkb) [inline plugins.json:2382]:
```suggestion
            "summary": "Gruvbox theme for (Neo)vim with the Material Palette",
```

**Review by mrcjkb — CHANGES_REQUESTED** [review]

**Review by mrcjkb — APPROVED** [review]

**Comment by adam-devel** [comment]
forgot to pull before rebasing, re-fixed the description

**Comment by adam-devel** [comment]
If I have a large set set of plugins to add, would it be more appropriate to add them in one single pull request to avoid rebasing in between merges ?

**Comment by mrcjkb** [comment]
> If I have a large set of plugins to add, would it be more appropriate to add them in one single pull request to avoid rebasing in between merges ?

Yes, that should be fine 😃

**Pull request merged**